shared/gh/js/controllers/gh.api.util.js

```diff
@@ -437,6 +437,47 @@ define(['exports', 'moment', 'bootstrap-notify'], function(exports, moment) {
     };
 
     /**
+     * Order the terms and split up the out of term events
+     *
+     * @param  {terms}     terms    The terms that should be reordered
+     * @return {Object}             Object containing the reordered terms and events
+     */
+    var orderEventsByTerm = exports.orderEventsByTerm = function(terms) {
```

Comment:
As discussed, I will leave this functionality in here. I will add some more function descriptions.